guides/release/tutorial/deploying.md

```diff
@@ -32,6 +32,34 @@ You can deploy your application to any web server by copying the output from `em
 scp -r dist/* myserver.com:/var/www/public/
 ```
 
+## Deploying to ZEIT Now
```

Comment:
- Change the level 2 heading to a level 3 heading
- change the text to ZEIT Now
- place in alphabetical order with the other options

Reply:
Done